include/ts/remap_version.h

```diff
@@ -43,5 +43,5 @@
   } while (0)
 
 #ifndef ATS_BUILD
```

Comment:
Could remove the ATS_BUILD lines as well

Reply:
I was keeping it as a placeholder for when I add the namespaces back

Reply:
Still I will remove that...